packages/mobile-wallet/src/components/App/AddFunds/styles.js

```diff
@@ -0,0 +1,131 @@
+import React, { Fragment } from 'react';
+import styled from 'styled-components';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+import Button from 'components/shared/Button';
+import { colors, fonts } from 'components/shared/Styleguide';
+
+export const Container = styled.View`
+  flex: 1;
+  background-color: ${colors.bgLight};
+`;
+
+export const Touchable = styled.TouchableHighlight``;
+
+export const SelectorContainer = styled.View`
+  position: relative;
+  padding-top: 18;
+  padding-left: 17;
+  padding-right: 17;
+  padding-bottom: 18;
+  border-bottom-width: 1;
+  border-color: #e1e1e1;
```

Comment:
let's replace `colors.borderColor` (or `colors.lightBorder` - depending on brightness) value with this one - seems like this will be "default" border color on another screens